app/client/components/helpCentre/helpCentreLandingMoreTopics.tsx

```diff
@@ -0,0 +1,95 @@
+import { css } from "@emotion/core";
+import { space } from "@guardian/src-foundations";
+import { textSans } from "@guardian/src-foundations/typography";
+import React, { useState } from "react";
+import { trackEvent } from "../analytics";
+import { helpCentreMoreQuestionsConfig } from "./helpCentreConfig";
+import {
+  containterCss,
+  innerSectionCss,
+  innerSectionDivCss,
+  linkAnchorStyle,
+  linkArrowStyle,
+  sectionTitleCss
+} from "./helpCentreStyles";
+
+const moreTopicsStyles = css({
+  marginBottom: "10px",
+  display: "flex",
+  flexWrap: "wrap",
+  textAlign: "left",
+  fontWeight: "normal"
+});
+
+export const HelpCentreLandingMoreTopics = () => {
+  const [indexOfOpenSection, setIndexOfOpenSection] = useState<number>();
+
+  const showHideCss = `
+    ${textSans.xsmall()};
+    margin-left: ${space[3]}px;
+    @media only screen and (max-width: 375px) {
+    display: none;
+  }
```

Comment:
```suggestion
    ${maxWidth.mobileMedium} {
       display: none
     }
```

Reply:
> 

we've got some breakpoints setup already in `/client//styles/breakpoints.ts` that we can use for this :)